test-apps/appui-test-app/standalone/src/frontend/appui/BlenkConnection.ts

```diff
@@ -0,0 +1,65 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
Typo in the name of this file `BlankConnection.ts` (a instead of e)

Reply:
Good catch, fixed.